src/ngraph/op/pad.cpp

```diff
@@ -174,3 +174,97 @@ std::shared_ptr<Node> op::Pad::get_default_value() const
     }
     return std::make_shared<op::Broadcast>(input_value(1), get_shape(), axes);
 }
+
+const string op::v1::Pad::type_name{"Pad"};
+
+op::v1::Pad::Pad(const Output<Node>& arg,
+                 const Output<Node>& pads_begin,
+                 const Output<Node>& pads_end,
+                 const Output<Node>& arg_pad_value,
```

Comment:
Any reason for this node ordering? arg, arg_pad_value, pads_begin, pads_end would be more consistent with v0.

Reply:
I've thought about it. Such ordering is defined in `Pad` specification. I am not sure if we have to keep specification order or be consistent with `v:0`. @ilyachur @GlebKazantaev What do you think about it?

Reply:
Arguments order the same as in spec and that's right.